src/init.cpp

```diff
@@ -628,8 +619,8 @@ static void ZC_LoadParams()
 
     pzerocashParams = new libzerocash::ZerocashParams(
         INCREMENTAL_MERKLE_TREE_DEPTH,
-        &pk_loaded,
-        &vk_loaded
+        &vk_loaded,
+        pk_path.string()
```

Comment:
It seems odd for the pk and vk-related parameters to be in a different order for the two constructors.